documentation/book/ref-storage-jbod.adoc

```diff
@@ -3,20 +3,26 @@
 // assembly-storage.adoc
 
 [id='ref-jbod-storage-{context}']
-= JBOD storage
+= JBOD storage overview
 
-The JBOD storage allows you to provide more volumes to the Kafka brokers to store data.
-As well as providing more storage, using multiple disks can improve performance.
+You can configure {ProductName} to use Just a Bunch of Disks (JBOD), a data storage configuration in which multiple disks are combined to create a single logical disk, or volume. JBOD is one approach to providing increased data storage for Kafka brokers. It can also improve performance.
 
-To use JBOD storage, the `type` has to be set to `jbod`.
-A JBOD storage is described by one or more volumes and each of them can be an xref:ref-ephemeral-storage-{context}[Ephemeral] or a xref:ref-persistent-storage-{context}[Persistent] one.
-Each volume declaration follows the same rules and constraints as for Ephemeral and Persistent storage.
-For example, it is not possible to change the size for a Persistent storage when it is already provisioned.
-The JBOD storage provides a way for putting together more of them.
+A JBOD configuration is described by one or more volumes, each of which can be either xref:ref-ephemeral-storage-{context}[ephemeral] or xref:ref-persistent-storage-{context}[persistent]. The rules and constraints for JBOD volume declarations are the same as those for ephemeral and persistent storage. For example, you cannot change the size of a persistent storage volume after it has been provisioned.
 
-Using the `volumes` property, it is possible to declare the disks which make the JBOD storage.
+== JBOD configuration
+
+To use JBOD with {ProductName}, the storage `type` must be set to `jbod`. 
+
+[source,yaml]
+----
+# ...
+storage:
+  type: jbod
+# ...
+----
```

Comment:
I don't see any reason for having this "empty" example just setting the storage `jbod` type without volumes, because it cannot work.

Reply:
Hmm, I see your point. I just wanted to call out the relevant type. I'll remove it.